src/client/utils/rewriteImports.ts

```diff
@@ -16,10 +16,10 @@ function alias(key: string): {key: string; name: string} {
 let num: number;
 function generate(keys: string[], dep: string, base: string, fn: string): string {
 	const tmp =
-		(dep
+		dep
```

Comment:
I think it can be done simpler and even more bulletrproof:

```suggestion
		dep.replace(/\W/g, '_')
```

You don't need split, map and join here.

Reply:
Excellent...